packages/phrases-experience/src/locales/zh-cn/description.ts

```diff
@@ -64,23 +64,23 @@ const description = {
     '你的企业已为电子邮件账户{{email}}启用了单点登录。你可以继续使用以下SSO提供商进行登录。',
   single_sign_on_enabled: '该帐户已启用单点登录',
   /** UNTRANSLATED */
```

Comment:
These flags should be removed since translations are added.

Reply:
I will fix that, but I find pushing to the master is bad way, so, I will close this, and pulling a new requested

Reply:
We generally insert a space between full-width characters (such as Chinese, Japanese and Korean characters), and half-width English letters, Arabic numerals, and symbols, to provide a better reading experience.